### PR TITLE
FAI-11604: Pull Request Comments source + converter

### PR DESCRIPTION
**Pull request by matiaslcoulougian**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: sources/github-source/test/resources/pull_request_comments/pull_request_comments.json [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    "original_start_line": 12,
+    "start_side": "RIGHT",
+    "line": null,
+    "original_line": 12,
+    "side": "RIGHT",
+    "original_position": 33,
+    "position": null,
+    "subject_type": "line"
+  }
+]
```
Comment (chalenge) [inline sources/github-source/test/resources/pull_request_comments/pull_request_comments.json:114]: nit: newline

**Review by chalenge — APPROVED** [review]
:shipit:

File: sources/github-source/test/index.test.ts [hunk 2]
```diff
@@ -408,6 +434,18 @@ const getPullRequestsMockedImplementation = (res: any) =>
 const getLabelsMockedImplementation = (res: any) =>
   graphqlMockedImplementation('labels', res);
 
+const getPullRequestCommentsMockedImplementation = (res: any) => ({
+  pulls: {
+    listReviewCommentsForRepo: jest
+      .fn()
+      .mockReturnValue(
```
Comment (dbruno21) [inline sources/github-source/test/index.test.ts:441]:
```suggestion
      .mockReturnValue(res),
```

Comment (dbruno21) [inline sources/github-source/test/index.test.ts:441]: .mockReturnValue(res)

**Pull request merged**